ballerina/types.bal

```diff
@@ -56,3 +56,25 @@ public type CorsConfig record {|
 isolated service class HttpService {
     *http:Service;
 }
+
+// GraphQL client related data binding types representation
+
+# Represents the target type binding record with data and extensions of a GraphQL response for `executeWithType` method.
+#
+# + extensions -  Meta information of the GraphQL API call
+# + data -  Data information of the GraphQL API call
```

Comment:
```suggestion
# + data -  The requested data from the GraphQL server
```

Reply:
Replaced in [f26678c](https://github.com/ballerina-platform/module-ballerina-graphql/pull/663/commits/f26678c4ceef61a6ce293851142e167fd6e26c07). Thank you.